src/ui-kit/form-controls/autocomplete/autocomplete.component.ts

```diff
@@ -71,6 +71,10 @@ export class SamAutocompleteComponent
   * Sets the hint text
   */
   @Input() public hint: string;
+  /**
+   * set to false if more/less is not required
+   */
+  @Input() public hideShowMore :  boolean = true;
```

Comment:
should this be called `showShowMore` if default value is `true`? I think we should maybe use a name like `hintToggler` (these seem to be really important to get right in case we do major component changes in the future).

Reply:
@diego-ruiz Good point. I think `hintToggler` would be a good name for a class that dealt with the toggling logic. For just a feature flag, maybe `toggleHint` with a default value of false?

Reply:
Or @diego-ruiz `hideHint` with default value `false`?

Reply:
suggestion : hideFullHintText or isTruncateHint or isTruncateHintText ? @colinalford-gsa @diego-ruiz

Reply:
would like to keep the name on the simpler side if possible. `toggleHint` seems okay

Reply:
One thing I don't like about toggle, it doesn't tell you anything about the default state. I think `toggle` is a good word to use if you want to switch to the opposite of two binary states regardless of the current state. Our case is kind of like `disabled` in html -- it always has a default value and only changes if provided a value. I think `hideHint` is a little better since it is explicit about what will happen when you set the value to true, and if you don't want to touch anything, you don't provide a value at all.